sdk/ai/azure-ai-inference/azure/ai/inference/models/_models.py

```diff
@@ -957,7 +1026,7 @@ class FunctionDefinition(_model_base.Model):
      interpreting its parameters.
     :vartype description: str
     :ivar parameters: The parameters the function accepts, described as a JSON Schema object.
-    :vartype parameters: any
+    :vartype parameters: dict[str, any]
     """
```

Comment:
You'll need to apply the above change in the public hand-written class DeveloperMessage in _patch.py. And similarly for the other XxxxMessage classes.